scripts/deploy-docs.sh

```diff
@@ -9,21 +9,15 @@ fi
 
 pip3 install --upgrade pip
 pip3 install --user --requirement <(cat <<EOF
-Click==7.0
 future==0.18.2
```

Comment:
All these were added as a poor man's lock file. Now the dependencies will change over time

Reply:
@Jellyfrog these were dependencies of the old windmill theme.  (I'm 90% sure).   I did a clean env and installed the top level items and "locked them to that version".